client-js/app/elements/labrad-nodes.html

```diff
@@ -180,9 +190,40 @@
     td.controls {
       width: 350px;
     }
+    #buttons {
+      height: 50px;
+    }
+    #buttons paper-icon-button {
+      margin: 5px;
+    }
+    .autostartFilterOn {
+      color: #000;
+    }
+    .autostartFilterOff {
+      color: #aaa;
+    }
   </style>
   <template>
-    <div>
+   <div id="container" class='fit'>
+   <paper-header-panel id="left-column">
+    <div class="paper-header" id="buttons">
+      <paper-icon-button
+        hidden$={{isAutostartFiltered}}
+        class='autostartFilterOff'
```

Comment:
Yeah, my brain struggles with the swapping between TypeScript (single) and HTML (double). I was actually thinking to mention we may want to standardise HTML to single quotes to make it more consistent. I do believe the style guide allows either as long as it's consistent.